src/transforms/__tests__/textShadow.js

```diff
@@ -0,0 +1,38 @@
+import transformCss from '../..'
+
+it('textShadow with all values', () => {
+  expect(transformCss([['text-shadow', '10px 20px 30px red']])).toEqual({
+    textShadowOffset: { width: 10, height: 20 },
+    textShadowRadius: 30,
+    textShadowColor: 'red',
+  })
+})
+
+it('textShadow omitting blur', () => {
+  expect(transformCss([['text-shadow', '10px 20px red']])).toEqual({
+    textShadowOffset: { width: 10, height: 20 },
+    textShadowRadius: 0,
+    textShadowColor: 'red',
+  })
+})
+
+it('textShadow omitting color', () => {
+  expect(transformCss([['text-shadow', '10px 20px black']])).toEqual({
+    textShadowOffset: { width: 10, height: 20 },
+    textShadowRadius: 0,
+    textShadowColor: 'black',
+  })
+})
+
+it('textShadow omitting blur, offset-y', () => {
+  expect(transformCss([['text-shadow', '10px 20px red']])).toEqual({
```

Comment:
also this tests seems to be the same as previous ones.

Reply:
Another good catch, this should be removed